scripts/glv_lattice_basis/src/arithmetic.rs

```diff
@@ -0,0 +1,34 @@
+use algebra_core::biginteger::BigInteger;
```

Comment:
Seems like GLV stuff did sneak in :(
Can you remove?

Also all the GLV bools.